xcodeproj/internal/xcodeproj_runner.bzl

```diff
@@ -557,6 +558,7 @@ xcodeproj_runner = rule(
         "pre_build": attr.string(),
         "project_name": attr.string(mandatory = True),
         "project_options": attr.string_dict(mandatory = True),
+        "scheme_autogeneration_config": attr.string_list_dict(),
```

Comment:
```suggestion
        "scheme_autogeneration_config": attr.string_dict(mandatory = True),
```

Reply:
@brentleyjones mind clarifying the intent here? If we don't pass a `string_list_dict` here we can't propagated a `list` of patterns in a single key (i.e. `scheme_name_exclude_patterns`).

Reply:
Yeah, just apply the `mandatory` for now. I was prematurely trying to account for when we have values that aren’t lists. I think when we get to that we can just json encode the full thing.

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3027/commits/8ae2d3b87f439c72c1d42f2c1999216bf706d042